Makefile

```diff
@@ -8,7 +8,7 @@ all: build docker
 
 build: test
 	GOOS=linux go build -v -o ./bin/${BINARY_NAME} ./cmd/main.go
-	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-w -s' -a -installsuffix cgo -o ./bin/sleep ./sleep/sleep.go
```

Comment:
I took it out because I care about the container image build, not the local binary build... the former guarantees the correct env for the build; the assumes the user figured out how to install the correct golang deps. 

This seems to be the only Che repo using a makefile; perhaps if @ibuziuk agrees we could trash it and simplify by having ONLY the rhel8-based dockerfile instead of 3 different ways to build the same thing.